draft-ietf-quic-version-negotiation.md

```diff
@@ -68,6 +68,75 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 
+# Server Deployments of QUIC {#server-fleet}
+
+While this document mainly discusses a single QUIC server, it is common for
+deployments of QUIC servers to include a fleet of multiple server instances. We
+therefore define the following terms:
+
+Supported Versions:
+
+: This is the set of versions supported by a given individual server instance.
+
+Fully-Deployed Versions:
+
+: This is the set of QUIC versions that is supported by every single QUIC
+server instance in this deployment.
+
+Partially-Deployed Versions:
+
+: This is the set of versions supported by a given individual server instance,
+but which aren't supported by every single QUIC server instance in this
+deployment.
+
+If a deployment only contains a single server instance, then the Supported
+Versions is equal to the Fully-Deployed Versions, and the Partially-Deployed
```

Comment:
nit: "Supported Versions" is plural, but "is" is singular. You might consider using "are" instead of "is" or alternatively dodging by using "Supported Versions set" (which is singular).